src/styles/sidebar/components/markdown-editor.scss

```diff
@@ -0,0 +1,44 @@
+$toolbar-border: 0.1em solid $grey-3;
+
+.markdown-editor__toolbar {
+  display: flex;
+  flex-direction: row;
+
+  background-color: $white;
+  border-top: $toolbar-border;
+  border-left: $toolbar-border;
+  border-right: $toolbar-border;
+  border-radius: 0.15em 0.15em 0 0;
```

Comment:
I happen to be highly pro `em` units, but am curious what led you to use `em`s instead of `px` here.

Reply:
This was copied from the old styling to preserve the look and feel.